subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/executer/OutputScrapingExecutionResult.java

```diff
@@ -217,6 +218,30 @@ public String getError() {
         return error.withNormalizedEol();
     }
 
+    @Override
+    public String getOutputLineThatContains(String text) {
+        String searchedText = LogContent.of(text).withNormalizedEol();
+        String actualText = getMainContent().withNormalizedEol();
```

Comment:
How about you replace all of this with something like:
```
getLines().stream()
            .filter(line -> line.contains(text))
            .findFirst();
```

Reply:
Indeed, much better!